.github/workflows/ci.yml

```diff
@@ -121,14 +121,16 @@ jobs:
     strategy:
       matrix:
         amiga:
-        # - { name: "AmigaOS 3", host: "m68k-amigaos", system: "amigaos3" }
-        - { name: "AmigaOS 4", host: "ppc-amigaos", system: "amiga" }
+        # - { name: "AmigaOS 3", host: "m68k-amigaos", cmake-flags: -DM68K_CRT=nix20 }
```

Comment:
Where is this M68K_CRT define used? (and the PPC one on the line below)

Reply:
They're used in the [toolchain files](https://github.com/AmigaPorts/AmigaCMakeCrossToolchains) that are bundled with the docker images.